From 090a9a1964f3477d86cfa1c5e2b4018c7c48b778 Mon Sep 17 00:00:00 2001
From: Alex Riesen <alexander.riesen@cetitec.com>
Date: Thu, 21 Dec 2017 09:29:39 +0100
Subject: [PATCH 142/454] Add Raspberry Pi firmware driver to the dependencies
 of backlight driver

Otherwise the backlight driver fails to build if the firmware
loading driver is not in the kernel

Signed-off-by: Alex Riesen <alexander.riesen@cetitec.com>
---
 drivers/video/backlight/Kconfig | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/video/backlight/Kconfig
+++ b/drivers/video/backlight/Kconfig
@@ -267,6 +267,7 @@ config BACKLIGHT_PWM
 
 config BACKLIGHT_RPI
 	tristate "Raspberry Pi display firmware driven backlight"
+	depends on RASPBERRYPI_FIRMWARE
 	help
 	  If you have the Raspberry Pi DSI touchscreen display, say Y to
 	  enable the mailbox-controlled backlight driver.
